app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CompaniesController < ApplicationController
```

Comment:
Now that I've fetched upstream I can't even run my test files anymore. I keep getting a "[Coveralls] Outside the CI environment, not sending data." error and the only solution I've found is to put 'Coveralls.wear!' in the spec helper but that didn't work - I'm still getting the same error and no tests are running.